Dev 3.22

GigaZiv opened the pull request:
Выполнена задача на 22 спринт

С уважением Жуков Игорь

GigaZiv committed:
Dev 3.22

Выполнена задача на 22 спринт

С уважением Жуков Игорь

kosstyango commented:
⚠️ Критические замечания:
   Рамки вокруг полей "Название" и "Описание" ДО установки в них курсора должны быть серые в светлой теме и белые - в темной теме (см.макет в Фигма), а не бирюзовые в обеих темах (см.скрин):
![Спринт№22_Жуков1](https://github.com/user-attachments/assets/61925eb5-1ecc-4b43-9a19-a50c6faf9545)

    При попытке добавить трек в плейлист, где такой трек уже есть, окно добавления НЕ должно исчезать.
    На экране Аудиоплеер вместо названия трека показывается слово "Заголовок"(см.скрин):
![Спринт№22_Жуков2](https://github.com/user-attachments/assets/c1b5451c-6da5-4fd7-b217-e2f545611999)

    При создании плейлиста из всплывающего окна добавления трека в плейлист, пользователь возвращается назад, но НЕ на всплывающее меню, а на экран "Аудиоплеер". Приходится снова нажимать на кнопку "добавить в плейлист" - это неправильно.
    При отмене создания плейлиста из всплывающего окна добавления трека в плейлист, пользователь возвращается не во всплывающее меню, а на экран "Аудиоплеер" (ещё и тост появляется "Трек уже добавлен...")

🍏 Рекомендации, не влияющие на результат проверки работы:
       Дефекты прошлых спринтов: курсор в поле поиска должен быть СИНИЙ, а не бирюзовый.
                                Прогресс-бар должен быть СИНИЙ, а не сиреневый.

GigaZiv committed:
Dev 3.22.1

1. Исправлены Рамки вокруг полей "Название" и "Описание" - приведены к требуемому цвету
2. Исправлено поведение - при попытке добавить трек в плейлист, где такой трек уже есть, окно добавления НЕ должно исчезать
3. Исправлено поведение при отмене создания плейлиста (появлялся тост)

От себя добавил debounce при добавлении трека в плай-лист + при нажатии на элемент поддерживающего состояние выделения для визуализации

С уважением Жуков Игорь

kosstyango commented:
Привет,
Ты поправил почти все "криты", молодец!
Остался последний:
При попытке добавить трек в плейлист, где такой трек уже есть, окно добавления НЕ должно исчезать. **Теперь окно НЕ исчезает даже при успешном добавлении трека, а кол-во треков не обновляется пока не переоткроешь всплывающее меню (см.видеоскрин):**
https://github.com/user-attachments/assets/e919e0bd-1dca-4c78-b03f-674e08626615


🍏 Рекомендации, не влияющие на результат проверки работы:
     При создании плейлиста из всплывающего окна добавления трека в плейлист, пользователь возвращается назад, но НЕ на всплывающее меню, а на экран "Аудиоплеер". Приходится снова нажимать на кнопку "добавить в плейлист" - **НЕкритично.**
Дефекты прошлых спринтов: курсор в поле поиска должен быть СИНИЙ, а не бирюзовый. **Это пока осталось**
                             Прогресс-бар должен быть СИНИЙ, а не сиреневый. **Это тоже пока осталось**

GigaZiv committed:
Dev 3.22.2

1. Исправлено поведение BottomSheet
2. Курсор и прогресс сделал синим

С уважением Жуков Игорь
